content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -5,5 +5,261 @@ weight: 2
+
+4. **TCP/IPv4 only:** In the current release, TCP over IPv4 is the only protocol supported for transport on an Istio secure overlay tunnel (this includes protocols such as HTTP that run between application layer endpoints on top of the TCP/ IPv4 connection).
+
+5. **No Dynamic switching to Ambient mode:** Enabling ambient mode is only possible during the deployment of a new Istio mesh control plane using an ambient profile or helm configuration. An existing Istio mesh deployed using a pre-ambient profile for instance can not be dynamically switched to also enable ambient mode operation.
+
+6. **Restrictions with Istio `PeerAuthentication`:** as of the time of writing, the `PeerAuthentication` resource is not supported by all components (i.e. waypoint proxies) in Istio ambient mode. Hence it is recommended to only use the `STRICT` mTLS mode currently. Like many of the other alpha stage caveats, this shall be addressed as the feature moves toward beta status.
+
+6. **`PeerAuthentication` Limitations:** As of now, not all components (i.e. Waypoint proxies), support the `PeerAuthentication` resource in Istio Ambient mode. Hence it is recommended to use the `STRICT` mTLS mode currently, this caveat shall be addressed as the feature moves toward beta status.
+
+7. **istioctl CLI Gaps:** Minor functional gaps may exist in the Istio CLI's output displays related to ambient mode. These will be addressed as the feature matures.
+
+In addition to this general caveats, there are also some specific caveats to be aware of when using Waypoint with certain protocols:
+
+- Waypoint only supports Ambient workloads. It does not support sidecar proxy workloads. 
+- HTTP: Waypoint does not support all HTTP features, such as HTTP/2 and chunked encoding.
+- gRPC: Waypoint does not support all gRPC features, such as HTTP/2 transport and protocol multiplexing.
+- WebSocket: Waypoint does not support WebSocket.
+
+Despite these caveats, Waypoint is a powerful tool for enabling L7 networking and services for Istio Ambient workloads. It is a good choice for users who are looking for a way to run microservices-based applications in Ambient mode.
+
+Here is a table summarizing the caveats of Waypoint:
+
+| Caveat | Description |
+| ------------- | ------------- |
+| Maturity | Waypoint is still under development |
+| Features | Waypoint only supports L7 load balancing and routing |
+|Integration | Waypoint is not yet fully integrated with the Istio control plane | Support | Waypoint only supports Ambient workloads | HTTP | Waypoint does not support all HTTP features | 
+gRPC | Waypoint does not support all gRPC features | Websocket | Waypoint does not support WebSocket |
```

Comment:
pending discussion from https://github.com/istio/istio.io/pull/13954/files#r1414064107, this table needs to be updated